BankApplication/src/com/luxoft/bankapp/tests/TestBankService.java

```diff
@@ -0,0 +1,181 @@
+package com.luxoft.bankapp.tests;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.Set;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+
+import com.luxoft.bankapp.handling_exceptions.ClientExistsException;
+import com.luxoft.bankapp.handling_exceptions.NotEnoughFundsException;
+import com.luxoft.bankapp.handling_exceptions.OverDraftLimitExceededException;
+import com.luxoft.bankapp.model.Account;
+import com.luxoft.bankapp.model.Bank;
+import com.luxoft.bankapp.model.CheckingAccount;
+import com.luxoft.bankapp.model.Client;
+import com.luxoft.bankapp.model.Gender;
+import com.luxoft.bankapp.model.SavingAccount;
+import com.luxoft.bankapp.service.BankServiceImpl;
+
+import junit.framework.Assert;
```

Comment:
This is outdated and deprecated class. You should use the org.junit.Assert, which you have imported above.